package.json

```diff
@@ -22,8 +22,8 @@
     "build:dist": "node bin/check-nvmrc.js && gulp build:dist --destination 'dist' && npm run test:build:dist",
     "test": "standard && gulp test && npm run test:app && npm run test:components && npm run test:generate:readme",
     "test:app": "jest app/__tests__/app.test.js --forceExit # express server fails to end process",
-    "test:components": "jest src/ && jest tasks/gulp/__tests__/check-individual-components-compile.test.js",
-    "test:generate:readme": "jest tasks/gulp/__tests__/check-generate-readme.test.js",
+    "test:components": "gulp copy-assets && jest src/ --forceExit && jest tasks/gulp/__tests__/check-individual-components-compile.test.js --forceExit",
```

Comment:
Could we document why we need to use `--forceExit` here?

Reply:
it is 'documented' on line 24. would be good to look into is and see if we can make it work without `--forceExit`, but maybe not part of this PR